qiskit/utils/optionals.py

```diff
@@ -13,8 +13,8 @@
 """
 .. currentmodule:: qiskit.utils.optionals
 
-Qiskit has several features that are enabled only
-if certain *optional* dependencies are satisfied.  This module is a collection of objects that can
+Qiskit has several features that are enabled only if certain *optional* dependencies
+are satisfied. The ``qiskit.utils.optionals`` module has a collection of objects that can
```

Comment:
```suggestion
are satisfied. This module, :mod:`qiskit.utils.optionals`, has a collection of objects that can
```

or something? - I'm not tied to the wording, I just want to make it clearer that this section specifically documents _one module_, and so the module name is a hyperlink anchor to the same place to drive the point home.

(This comment is predicated on the `` :mod:`qiskit.utils.optionals` `` link being removed from the title - if that's reinstated, I think the original wording of the docs are probably fine.)

Reply:
> and so the module name is a hyperlink anchor to the same place to drive the point home.

I thought this was confusing? I can add it back, but it seemed weird to me to click a link and it take you back to the exact spot you're reading.

Reply:
If nothing else, _not_ having the `:mod:` will mean that it gets styled very differently to most things documented objects - it's probably best to have it even if it's not all that useful.  We have loads of examples of classes referring to themselves already, and all the `:mod:` tags that appear in titles will also be hyperlinking to themselves.

(Personally I don't find it confusing - I quite like it when things like the Python docs do it (e.g. https://docs.python.org/3/library/unittest.html)).